src/constants/index.ts

```diff
@@ -0,0 +1,5 @@
+export const constants = {
+  urlReleaseNote: 'https://github.com/robertohuertasm/vscode-icons/blob/master/CHANGELOG.md',
```

Comment:
We should use the new url `https://github.com/vscode-icons/vscode-icons/blob/master/CHANGELOG.md`